src/lib/components/CopyCode/styles.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../../styles/settings/colors";
+@import '../../../styles/settings/global';
+
+.w-copy-code-button {
+  background: none;
+  background-image: url("../../../images/icons/file_copy.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  border: none;
+  border-radius: $GLOBAL_ROUNDED;
+  float: right;
+  height: 48px;
+  padding: 0;
+  width: 48px;
+  z-index: 100;
+  opacity: 0;
+
+  &:hover,
+  &:focus {
+    background-color: rgba($WHITE, 0.5);
```

Comment:
I think the white on the grey background is a little too low contrast. Maybe we can try one of the other colors to see if it provides a better visual. Maybe $WEB_SECONDARY_COLOR?

Reply:
I'm not sure if I love the color, so I lowered the opacity. I also applied a transition which smoothes out the animation.